src/server/components/dashboard/routes.ts

```diff
@@ -13,6 +13,6 @@ export const dashboardRoutes = {
   listsEdit: "/dashboard/lists/:listId",
   listsItems: "/dashboard/lists/:listId/items",
   listsItemsApprove: "/dashboard/lists/:listId/items/:listItemId/approve",
+  listsItemsDelete: "/dashboard/lists/:listId/items/:listItemId/delete",
```

Comment:
should this just be using the `delete` http verb on `/dashboard/lists/:listId/items/:listItemId`?

Reply:
In an ideal world yes, but the other endpoints have the verb on the URL so i've done the same to be consistent. Happy to make inconsistent - but more correct if you wish

Reply:
you mean like approve/publish? they're ok since they're not http verbs, but if we're updating/adding/deleting a resource we should do them on `POST | GET | DELETE /dashboard/lists/:listId/items/:listItemId`. (Unless the other endpoints use /post, /delete, /edit etc, then that's OK for consistency)